content/en/security_platform/guide/how-appsec-works.md

```diff
@@ -0,0 +1,51 @@
+---
+title: How Application Security works in Datadog
+kind: guide
+---
+
+## Overview
+
+Application Security monitors application-level attacks aiming to exploit code-level vulnerabilities.
+
+Application Security records information about each request and applies security rules in-app. This information enriches your APM traces with security data.
+
+Traditional Web Application Firewalls (WAFs) are usually deployed at the perimeter and have no context of the application behavior. Application Security provides WAF-like capabilities with additional application context and lower false positives.
+
+## Threat detection methods
+
+Datadog uses multiple pattern sources, including the [OWASP ModSecurity Core Rule Set][1] to detect known threats and vulnerabilities in HTTP requests. When an HTTP request matches one of those rules, a security signal is generated in Datadog.
+
+Signals are automatically created when Datadog detects meaningful attacks targeting your production services. It provides you with visibility on the attackers and the targeted services. You can set monitors with thresholds to determine which attacks you want to be notified about.
+
+## Attack types
+
+Application Security includes over 100 OOTB detection rules that help safegaurd against many different kinds of attacks, including the following:
+
+* SQL injections
+* Code Injections 
+* Shell Injections
+* NoSQL injections 
+* Cross-Site Scripting (XSS) 
+* Sever-side Request Forgery (SSRF)
```

Comment:
Full source, see the type facet in the AppSec configuration page https://app.datadoghq.com/security/appsec/event-rules

Feel free to link to this